### ISSUE TITLE
connect Google AdWords to gridcoin.us analytics

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#34]
goal: to get better info

further info:
* [Gridcoin Advertising (2): Expanding our Google AdWords budget](https://steemit.com/gridcoin/@vortac/gridcoin-advertsing-2-expanding-our-google-adwords-budget)
* [poll: new campaign Google AdWords (2017 Jan 1 - Dec 31), $10 daily, paid via foundation funds](https://steemit.com/gridcoin/@erkan/gridcoin-poll-new-campaign-google-adwords-2017-jan-1-dec-31-usd10-daily-paid-via-foundation-funds-vote-until-january-28)

task: @grctest + Vortac

mentioned in [hangout 22](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2017_01_21.MD):

**Comment by grctest** [comment]
I've contacted vortac, we're going to try linking analytics:adwords for reviewing the effectiveness of adwords campaigns.

**Comment by Vortac2** [comment]
That's an excellent idea. I tried to judge the Adwords effectiveness by monitoring the influx of new team members on BOINCstats, but that's far from reliable. 
Another way is to monitor 'users by source' through statsbot on TeamGridcoin Slack. Users coming through Adwords click aggregators appear regularly there and number of users coming through Google has also increased since we activated our keywords on Google searches, but it's still difficult to tell how effective this effort truly is.
